### PR TITLE
feat: add file and upload icons

### PR DESCRIPTION
**Pull request by ganiirsyadi**

### CODE REVIEW

File: src/@next/Icon/icons/ri-file-fill.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 24 24"
+  xmlns="http://www.w3.org/2000/svg">
```
Comment (ninariccimarie) [inline src/@next/Icon/icons/ri-file-fill.svg]: NIT: can you make it into 1 line?

Comment (ninariccimarie) [reply src/@next/Icon/icons/ri-file-fill.svg]: same comment for the other icons

Comment (ganiirsyadi) [reply src/@next/Icon/icons/ri-file-fill.svg]: Ah, it was caused by my XML formatter. Let me fix it.

Comment (ganiirsyadi) [reply src/@next/Icon/icons/ri-file-fill.svg]: Fixed it

**Review by ninariccimarie — APPROVED** [review]
LGTM 🚀

**Pull request merged**